trackier-ios-sdk/Classes/Constants.swift

```diff
@@ -15,6 +15,7 @@ class Constants {
     static let INSTALL_URL = BASE_URL + "/install"
```

Comment:
need to update the version of the SDK as well @IloveJavaa

Reply:
done